libp2p/src/tutorials/hole_punching.rs

```diff
@@ -54,16 +54,17 @@
 //!
 //! ``` bash
 //! ## Inside the rust-libp2p repository.
-//! cargo build --example relay_v2 -p libp2p-relay
+//! cd ./examples/relay-server
+//! cargo build
```

Comment:
I think we can simplify this a bit so users don't have to change directories :)

```suggestion
//! cargo build --bin relay-server-example
```